state/migrations/package_test.go

```diff
@@ -9,11 +9,12 @@ import (
 	gc "gopkg.in/check.v1"
 )
 
+//go:generate mockgen -package migrations -destination externalcontroller_mock_test.go github.com/juju/juju/state/migrations MigrationExternalController,ExternalControllerSource,ExternalControllerModel
 //go:generate mockgen -package migrations -destination offerconntections_mock_test.go github.com/juju/juju/state/migrations MigrationOfferConnection,AllOfferConnectionSource,OfferConnectionSource,OfferConnectionModel
 //go:generate mockgen -package migrations -destination relationetworks_mock_test.go github.com/juju/juju/state/migrations MigrationRelationNetworks,RelationNetworksSource,RelationNetworksModel
 //go:generate mockgen -package migrations -destination remoteapplications_mock_test.go github.com/juju/juju/state/migrations MigrationRemoteApplication,AllRemoteApplicationSource,StatusSource,RemoteApplicationSource,RemoteApplicationModel
 //go:generate mockgen -package migrations -destination remoteentities_mock_test.go github.com/juju/juju/state/migrations MigrationRemoteEntity,RemoteEntitiesSource,RemoteEntitiesModel
-//go:generate mockgen -package migrations -destination description_mock_test.go github.com/juju/description OfferConnection,RemoteEntity,RelationNetwork,RemoteApplication,RemoteSpace,Status
+//go:generate mockgen -package migrations -destination description_mock_test.go github.com/juju/description ExternalController,OfferConnection,RemoteEntity,RelationNetwork,RemoteApplication,RemoteSpace,Status
```

Comment:
I realized some patches ago that we tend to do that. 
Seems like a good thing to me, do we that pattern documented somewhere?
Adding the `go:generate` commands in the respective `package_test.go` file?

Reply:
https://discourse.jujucharms.com/t/go-generate-location/2436